src/EditorFeatures/CSharpTest/CodeActions/SyncNamespace/SyncNamespaceTests.ChangeNamespaceTests.cs

```diff
@@ -0,0 +1,1328 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeActions.SyncNamespace
+{
+    public partial class SyncNamespaceTests : CSharpSyncNamespaceTestsBase
+    {
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsSyncNamespace)]
+        public async Task ChangeNamespace_InvalidFolderName1()
```

Comment:
Same for this file. I think `Theory` and `InlineData` will really help make your tests lot more readable and succinct.